backend/src/server.ts

```
@@ -89,11 +89,73 @@ export async function createApp() {
+   *  4. Requests with no Origin header (curl, server-to-server) are allowed.
+   *  5. Requests whose Origin matches the whitelist receive a reflected
+   *     `Access-Control-Allow-Origin` + `Access-Control-Allow-Credentials: true`.
+   *  6. All other origins are rejected (CORS preflight returns 415).
+   */
+  const allowedOrigins = (() => {
+    const origins: string[] = [];
+
+    // Explicitly configured whitelist
+    if (process.env.CORS_ORIGIN) {
+      origins.push(
+        ...process.env.CORS_ORIGIN.split(',').map((o) => o.trim()).filter(Boolean)
```

ESLint

[error] Replace `.map((o)·=>·o.trim())` with `⏎··········.map((o)·=>·o.trim())⏎··········`

(prettier/prettier)